### ISSUE TITLE
Upgrade the  org.eclipse.persistence.jpa.jpql plugin version

### ISSUE THREAD

**Issue by lak-proddev** [eclipse-dali/webtools.dali#9]
It's required to update the JPQLGrammar for JPA 3.0 and  3.1 support.

https://bugs.eclipse.org/bugs/show_bug.cgi?id=579533#c5
> _I suppose the Tycho support for Maven artifacts would let use use hhttps://search.maven.org/artifact/org.eclipse.persistence/org.eclipse.persistence.jpa.jpql/3.0.2/jar directly?_

https://github.com/eclipse-ee4j/eclipselink/issues/1387

**Comment by lak-proddev** [comment]
@nitind  Don't merge the above commit for some time. Need to do more testing before merging.

The #9 and #10 are potentially risk fixes for 2024-06, We will postpone to 2024-09 if time does not permit.
